Tutorials/02-Blazor/02C-MiddlewareTutorial/MiddlewareTutorial/Client/Store/WeatherUseCase/Effects.cs

```diff
@@ -16,9 +16,12 @@ public Effects(HttpClient http)
 		}
 
 		[EffectMethod]
-		public async Task HandleFetchDataAction(FetchDataAction action, IDispatcher dispatcher)
-		{
-			var forecasts = await Http.GetJsonAsync<WeatherForecast[]>("WeatherForecast");
```

Comment:
Http.GetJsonAsync gives a compile error. Had to use Get**From**JsonAsync instead